### PR TITLE
remove incorrect note about scheduled import

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
according to @smarterclayton this note isn't accurate (scheduled import is *enabled* by default).  That matches what i see when i do an openshift start --write-config (the config contains disabled=false).

@soltysh @mfojtik ptal/confirm.

File: dev_guide/managing_images.adoc [hunk 1]
```diff
@@ -826,12 +826,6 @@ spec:
       scheduled: true
 ----
 
-[NOTE]
-====
-By default, `*importPolicy.scheduled*` is disabled and must be enabled in the
```
Comment (soltysh) [inline dev_guide/managing_images.adoc]: Wait, there are two things mixed here. One is `importPolicy.scheduled` which is set *on an Image Stream*, per-tag to be honest. And second is the master config option `disableScheduledImport` which indeed is `false` by default. I'd suggest changing this description to explicitly mention that the IST's `scheduled` is disabled, and needs to be enabled explicitly.

**Review by soltysh — CHANGES_REQUESTED** [review]

**Comment by soltysh** [comment]
I've just spoke with @bparees on IRC, give the context of this change, this LGTM.

**Review by soltysh — APPROVED** [review]

**Comment by bparees** [comment]
@openshift/team-documentation got confirmation from @soltysh on irc, please merge.

**Comment by bmcelvee** [comment]
No revision history needed.

**Pull request merged**